Ci integration docs (#574)
alantech/alan: anycloud/docs/how-to/ci-integration.md

> AnyCloud CLI has enable a set of flags in order to deploy applications directly from a CI/CD tool in a non-interactive way.

Reviewer: ```suggestion
AnyCloud CLI has a set of flags useful for deploying applications directly from a CI/CD tool.
```
